models/order/order.validation.js

```diff
@@ -0,0 +1,32 @@
+import Joi from 'joi';
+import { isObjectId } from '../../helpers/index.js';
+import { orderStatus } from '../../constants/index.js';
+
+const idValidationSchema = Joi.string().custom(isObjectId, 'Invalid id');
+
+export const orderItemValidationSchema = Joi.object().keys({
+  dishId: idValidationSchema.required(),
+  count: Joi.number().min(1).required(),
+  price: Joi.number().min(0.01).required(),
+});
+
+export const addressValidationSchema = Joi.object().keys({
+  country: Joi.string().required(),
+  city: Joi.string().required(),
+  street: Joi.string().required(),
+  coordinate: Joi.object().keys({
```

Comment:
Maybe it would be better, if we had more strict validation with some borders like from -90 to 90 for latitutde and from -180  180 for longitude, but it's up to you.

Reply:
Ok. Done.